config/main.py

```diff
@@ -4513,6 +4513,13 @@ def ip(ctx):
     """Set IP interface attributes"""
     pass
 
+def validate_vlan_format(text):
+    pattern = re.compile(r'^Vlan([1-9]\d{0,2}|[1-3]\d{3}|40[0-8][0-9]|409[0-4])$')
+
+    if pattern.match(text):
```

Comment:
_return pattern.fullmatch(text) is not None_

I would use fullmatch instead of match, and remove the '^' and '$' (for simplicity's sake). because match returns an object only if it matches from the beginning of the string.
ex:
re.match('pattern', 'pattern in string') will match
re.fullmatch('pattern', 'pattern in string') will NOT match

Reply:
Thanks